lib/standard/bytes.nit

```diff
@@ -96,6 +97,16 @@ class Bytes
 		end
 	end
 
+	#     var b = new Bytes.empty
+	#     b.append([0x41u8, 0x41u8, 0x18u8])
+	#     b.pop
+	#     assert b.to_s == "AA"
+	redef fun pop do
+		assert length >= 1
```

Comment:
That would work too yes, should I change it or do I have permission to leave it as is ?